src/file.coffee

```diff
@@ -28,7 +28,7 @@ class File
   Section: Construction
   ###
 
-  # Public: Creates a new file.
+  # Public: Configures a new File instance, no files are accessed.
```

Comment:
As such, the documentation for `Directory::getFile()` appears to be inaccurate:

```
  # Public: Traverse within this Directory to a child File. This method doesn't
  # actually check to see if the File exists, it just creates the File object.
  #
  # * `filename` The {String} name of a File within this Directory.
  #
  # Returns a {File}.
  getFile: (filename...) ->
    new File(path.join @getPath(), filename...)
```

Because it calls `new File`, it does check to see whether the file path exists.